packages/geo-diary-marketing/src/components/intro-slide.svelte

```diff
@@ -8,22 +8,24 @@
       <div class="column el-flex-container el-flex-column el-flex-justify-evenly el-flex-grow no-margin">
         <img class="logo-image" src="./images/intro-title.svg" alt="Geo Diary logo" />
         <div class="el-mb6">
-          <h6 class="el-subtitle">Lets get started!</h6>
+          <h6 class="el-subtitle">Access your appointments, anywhere!</h6>
           <p class="el-body-text el-has-grey-text">
-            You can also use the listing preview to see how your app will display in the Marketplace to potential
-            clients.
+            Simple and easy to use. The GEO Diary app allows you to easily access your diary appointments dire¬ct from
```

Comment:
```suggestion
            Simple and easy to use. The GEO Diary app allows you to easily access your diary appointments direct from
```

Is this simple a tab or new line? Looks like a tab to me!

Reply:
symbol, jesus I was asleep